_source/user-guide/alerts/configure-alerts.md

```diff
@@ -151,19 +151,20 @@ You can search triggered alert logs at any time.
 
 If you chose recipients, you can also set the output format.
 
-![Output table](https://dytvr9ot2sszz.cloudfront.net/logz-docs/alerts/output-table.png)
+![Output table](https://dytvr9ot2sszz.cloudfront.net/logz-docs/alerts/output-json-custom-fields.png)
 
-You can choose to send **All fields** in JSON format,
-or **Custom fields** as a table.
+Choose whether to send the data as **JSON** or in a **Table**.
```

Comment:
My English is not as good as yours but still, I'll try... you use or to give two options, is it correct that one of them is JSON format and the other is a location(in a table). If the label is `Output format` isn't it suppose to say `...as a JSON or a Table`?

Reply:
In addition, let's make sure they understand it's a sample data no?

Reply:
"a JSON", not really - "a JSON object", maybe. I'll keep that part as is - but definitely yes on reminding the user it's a sample. That'll be in my next push to this PR.